### PR TITLE
Bump version

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request bumps the version so that a fresh AMI can be built.  It also bumps up the size of the filesystem used to build the AMI, since it was filling up and preventing the AMI from building successfully.

## 💭 Motivation and context ##

We want to build a fresh AMI to pick up changes from #67 as well as any system package updates.

## 🧪 Testing ##

I successfully built a new AMI for our COOL staging environment.

## ✅ Pre-approval checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All new and existing tests pass.
* [x] Build a staging AMI.

## ✅ Pre-merge checklist ##
* [x] Bump build in version.
* [x] Build a production AMI

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
👍 👍

File: src/packer.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
           "delete_on_termination": true,
           "device_name": "/dev/sda1",
           "encrypted": true,
-          "volume_size": 20,
+          "volume_size": 30,
```
Comment (mcdonnnj) [inline src/packer.json:9]: I see the commit about this, but I would like you to add a mention for this in the PR description if you don't mind.

**Review by mcdonnnj — APPROVED** [review]
LGTM ✔ I did have a request about documenting a change.

File: src/packer.json (hunk 1 above)
Comment (jsf9k) [reply src/packer.json:9]: Take a look now - I added a sentence or two.

Comment (mcdonnnj) [reply src/packer.json:9]: Perfect, thanks. I just wanted a mention for posterity 👍

**Pull request merged**